server/Aftok/Snaplet/Billing.hs

```diff
@@ -105,6 +109,23 @@ parseCreateBillable uid pid = unversion "Billable" p
         <*> (o .:? "paymentRequestMemoTemplate")
     p ver o = badVersion "Billable" ver o
 
+parseZcashChannelJSON :: Value -> Parser Zcash.Channel
```

Comment:
Does this duplicate what's in PostgreSQL/Json.hs?

Reply:
Hm, potentially. I usually like to keep persistence formats and RPC formats separate, but for trivial types there isn't as much value.